### PR TITLE
split usage

### PR DESCRIPTION
**Pull request by akash1810**
Splits usage to show `Preview` and `Published` content separately, also ensures video page usage is always at top of the list.

image

### CODE REVIEW

**Comment by akash1810** [comment]
[Separate pending and published usages](https://trello.com/c/9ydwIpHn/37-separate-pending-and-published-usages)

File: public/video-ui/src/components/VideoUsages/VideoUsages.js [hunk 1]
```diff
@@ -67,42 +61,50 @@ export default class VideoUsages extends React.Component {
...
+      <div className="form__group">
+        {this.renderUsages()}
+      </div>
+    );
   }
 }
 
 VideoUsages.propTypes = {
-  usages: PropTypes.array.isRequired,
+  // usages: PropTypes.object.isRequired,
```
Comment (mbarton) [inline public/video-ui/src/components/VideoUsages/VideoUsages.js]: Put back in?

Comment (akash1810) [inline public/video-ui/src/components/VideoUsages/VideoUsages.js]: oh yeah! I was having some problems with this... will check if I accidentally fixed it :)

**Comment by Reettaphant** [comment]
As we discussed, adding in a label after the video headline indicating its type might be helpful for users.

**Comment by akash1810** [comment]
The component is styled more subtly now:

image

**Pull request merged**